network-api/networkapi/mozfest/customblocks/stats_block.py

```diff
@@ -0,0 +1,15 @@
+from wagtail import blocks
+
+
+class StatisticCard(blocks.StructBlock):
+    title = blocks.CharBlock(help_text="The statistic figure, e.g '1000+' or '10%'")
+    description = blocks.CharBlock(help_text="Context or description for the statistic")
+
+
+class MozfestStatisticsBlock(blocks.StructBlock):
+    statistics = blocks.ListBlock(StatisticCard(), help_text="Please use a minimum of 2 cards.", max_num=4)
+
+    class Meta:
+        icon = "placeholder"
```

Comment:
Is there a nicer icon we could use? Something with a number maybe?

Reply:
There's no number icon, I guess table might make sense for a stat but it's a bit misleading itself.

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2128707/a555bd2a-395a-4869-88a4-e26706a74c51)